Modules/Options/TrackerTab/QuestieOptionsTracker.lua

```diff
@@ -133,28 +142,34 @@ function QuestieOptions.tabs.tracker:Initialize()
                 order = 1.7,
                 width = 1.0,
                 name = function() return l10n('Enable Header'); end,
-                desc = function() return l10n('When this is checked, the Questie Tracker Header will become visible and the total number of quests you have in your log will be shown.'); end,
+                desc = function() return l10n('When this is checked, the Active Quests Header will become visible and the total number of quests you have in your log will be shown.'); end,
                 disabled = function() return not Questie.db.global.trackerEnabled; end,
                 get = function() return Questie.db.global.trackerHeaderEnabled; end,
                 set = function(_, value)
                     Questie.db.global.trackerHeaderEnabled = value
                     QuestieTracker:Update()
+                    C_Timer.After(0.1, function()
+                        QuestieTracker:Update()
+                    end)
                 end
             },
             autoMoveHeader = {
                 type = "toggle",
                 order = 1.8,
                 width = 1.0,
                 name = function() return l10n('Auto Move Header'); end,
-                desc = function() return l10n("When this is checked, the Questie Tracker Header will automatically move to the bottom of the Questie Tracker while the Tracker is placed in the bottom area of your screen. Otherwise, it will remain at the top always.\n\nNOTE: This setting only works while the 'Tracker SetPoint' is using the default 'Auto' mode."); end,
+                desc = function() return l10n("When this is checked, the Active Quests Header will automatically move to the top or bottom of the Questie Tracker depending on which 'Tracked Quests Grow' setting is used.\n\nNOTE: This setting only works while the 'Tracked Quests Grow' setting is set to 'Up'."); end,
```

Comment:
The [translation](https://github.com/Questie/Questie/tree/master/Localization/Translations) needs to be fixed